### ISSUE TITLE
cant connect to my mi bedside lamp

### ISSUE THREAD

**Issue by andrii482** [vinsce/gneelight#2]
When I change my workround to kde  - I cant connect anymore to my bedside lamp. App is running in first case, but dont see lamp (lan control is enabled)

```
/usr/bin/flatpak run --branch=master --arch=x86_64 --command=runner.sh --devel -v me.vinsce.Gneelight
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/user216/.local/share/flatpak
F: Opening user flatpak installation at path /home/user216/.local/share/flatpak
F: Opening user flatpak installation at path /home/user216/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Cleaning up unused container id 329591372
F: Allocated instance id 2007594071
F: Add defaults in dir /me/vinsce/Gneelight/
F: Add locks in dir /me/vinsce/Gneelight/
F: Disallowing ipc access
F: Allowing wayland access
F: Allowing x11 access
F: Running 'bwrap --args 36 xdg-dbus-proxy --args=38'
F: Running 'bwrap --args 36 runner.sh'

```

```
python3 src/GneelightApp.py
Traceback (most recent call last):
  File "src/GneelightApp.py", line 7, in <module>
    from MainWindow import MainWindow
  File "/home/user216/gneelight/src/MainWindow.py", line 37
    self.bulb_wrapper: BulbWrapper = None
                     ^
SyntaxError: invalid syntax

```

**Comment by vinsce** [comment]
Hi @andrii482, which version of Python are you using?
Can you try with Python 3.6+?

**Comment by andrii482** [comment]
also looks like that you need to update flatpack:
```
flatpak install --user gneelight.flatpak

error: The application me.vinsce.Gneelight/x86_64/master requires the runtime org.gnome.Platform/x86_64/master which was not found
```

**Comment by vinsce** [comment]
> also looks like that you need to update flatpack:
> 
> ```
> flatpak install --user gneelight.flatpak
> 
> error: The application me.vinsce.Gneelight/x86_64/master requires the runtime org.gnome.Platform/x86_64/master which was not found
> ```

Hi @andrii482, this is a new problem and it is treated in issue #4.

If your original problem has been solved using Python 3.6+ we can close this issue.

**Comment by andrii482** [comment]
> Hi @andrii482, which version of Python are you using?
> Can you try with Python 3.6+?

Think that in init-message-issue I was used python 2.7. So I can close issue
upd. also I commented with new-problem-case in linked issue
